### ISSUE TITLE
Resolvers: Create tests for block_user.js

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa-api#578]
The Talawa-API code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file `lib/resolvers/block_user_mutations/block_user.js`

- We will need the API to be refactored for all methods, classes and/or functions found in this file for testing to be correctly executed.
- When complete, all all methods, classes and/or functions in the refactored file will need to be tested. These tests must be placed in a
single file with the name `tests/resolvers/block_user_mutations/block_user.spec.js`. You may need to create the appropriate directory structure to do this.

### IMPORTANT:
Please refer to the parent issue on how to implement these tests correctly:

- https://github.com/PalisadoesFoundation/talawa-api/issues/490

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- [The current code coverage for the file can be found here](https://codecov.io/gh/PalisadoesFoundation/talawa-api/tree/develop/lib/resolvers/block_user_mutations/). If the file isn't found in this directory, or there is a 404 error, then tests have not been created.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.

**Comment by ebrahim354** [comment]
can you assign this issue to me please.

**Comment by ebrahim354** [comment]
also there is already a file that tests block/unblock user called ./test/block_user_functionality_tests.spec.js

**Comment by palisadoes** [comment]
Please move the file to the expected location and if necessary, get it to 100% coverage.

**Comment by palisadoes** [comment]
We had issues with your PR. The test suite was breaking. Please take a look. We had to revert.

**Comment by ebrahim354** [comment]
it should be working now

**Comment by ebrahim354** [comment]
No, I have submitted a PR that covers all possible cases for this file and the unblock one

**Comment by palisadoes** [comment]
That PR had to be reverted as your tests were failing when others merged their code into the code base. You'll need to resubmit the PR.

**Comment by ebrahim354** [comment]
#630 was reverted then I resubmitted #646 and It was approved yesterday I think.

**Comment by palisadoes** [comment]
Thanks. Do you have any idea as to why the code coverage number for those files zero? 
- https://codecov.io/gh/PalisadoesFoundation/talawa-api/tree/develop/lib/resolvers/block_user_mutations/

**Comment by ebrahim354** [comment]
the tests are using axios and the main Api to test the resolvers from with in the apollo server which is the usual flow of the program in production, however the code coverage tool does not capture these tests instead counts when the file is directly accessed from the test file, if you want to see the actual coverage you should either try another tool (not sure this could work) or test the resolver file directly by importing it as I did on this PR: https://github.com/PalisadoesFoundation/talawa-api/pull/648 , but again you didn't specify how we should test the file exactly and there are also test on the repo that uses the api not the file directly the actual coverage should be higher than what is shown

**Comment by palisadoes** [comment]
I'm going to have to revert the code again. It's causing other tests to fail. 

- https://github.com/PalisadoesFoundation/talawa-api/runs/5759635905?check_suite_focus=true

Someone else is going to have to work on this.

**Comment by palisadoes** [comment]
Here is the error we are getting:

![image](https://user-images.githubusercontent.com/16875803/160909538-10a96568-4271-41bd-924b-d5545b30cdef.png)

**Comment by ebrahim354** [comment]
This issue should be resolved.

**Comment by SiddheshKukade** [comment]
All PRs are merged successfully.
